packages/backend/src/house/house.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Controller, Post, Body, Param, Get, Put, Query } from '@nestjs/common';
+import { UserStoreService } from '../db/user/userStore.service';
+import { HouseStoreService } from '../db/house/houseStore.service';
+import { CreateHouseDto } from './dto/create-house.dto';
+import { HouseUtil } from './house.util';
+
+@Controller('/api/v1/house')
```

Comment:
these should all be guarded
```suggestion
@Controller('/api/v1/house')
@UseGuards(FirebaseGuard)
```

Reply:
can you please also remove the guard from api.controller.ts